aquarius/events/metadata_updater.py

```diff
@@ -268,13 +277,16 @@ def get_datatoken_pools(self, dt_address, from_block=0, to_block='latest'):
             return None
 
         pools = [get_event_data(event_abi, log).address for log in logs]
-        return pools
+        return list(set(pools))
 
     def _get_liquidity_and_price(self, pools, dt_address):
         assert pools, f'pools should not be empty, got {pools}'
+        logger.debug(f' Searching {pools} for {dt_address}')
         dt_address_lower = dt_address.lower()
+        ocean_address_lower = self._OCEAN.lower()
```

Comment:
self._OCEAN is already lower case, to make this more clear, rename `self._OCEAN` to `self._ocean_lower_case` or so.